create-a-theme-checklist.js

```diff
@@ -30,69 +30,71 @@ async function createMilestone() {
 					'Automatically generated milestone for ' + theme + '.',
 			}
 		);
-	} catch ( error ) {
-		console.error( '\x1b[41m', 'Milestone already created.' );
+	} catch (error) {
+		console.error('\x1b[41m', 'Milestone already created.');
 	}
 }
 
-async function createIssues( milestoneNumber ) {
+async function createIssues(milestoneNumber) {
 	const issues = [
-		'Create base theme from boilerplate',
+		'Create base theme',
 		'Create a demo site on dotcom',
 		'theme.json: Typography settings',
 		'theme.json: Color palette',
 		'theme.json: Alternative color palettes',
```

Comment:
we probably don't need this at this point, since we are not including them, when we include theme.json variations we can revisit this